CSharpSDK/GetStores.cs

```diff
@@ -1,13 +1,52 @@
+using System;
+using System.Collections.Generic;
 using System.Net.Http;
 using System.Threading.Tasks;
 
 namespace CSharpSDK
 {
-    public partial class Killbills_sdk
+  public partial class Killbills_sdk
+  {
+    public class StoreData
+{
+    public List<object> Items { get; set; }
+}
+    public async Task<List<object>> GetStores(string env, string apiKey)
```

Comment:
no pagination ?

Reply:
Good point, but we didn't add pagination on any of the other packages